11/debian/buster/hotspot/Dockerfile

```
@@ -1,6 +1,22 @@
 FROM adoptopenjdk/openjdk11:jdk-11.0.11_9-debian
 
-RUN apt-get update && apt-get upgrade -y && apt-get install -y git curl && curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash && apt-get install -y git-lfs unzip && git lfs install && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y git curl unzip && \
```

The `gpg` program is needed for tini signature verification
```suggestion
    apt-get install -y git curl gpg unzip && \
```